lib/common.sh

```diff
@@ -449,3 +449,11 @@ function init_minikube() {
           --type network --config
     fi
 }
+
```

Comment:
this should probably go in upgrade common.sh if it is only used for the upgrade, no ?

Reply:
I would highly recommend to store the current value into some other variable, and restore it after finishing the test run

Reply:
the set_cap_variables is obsolete and to be removed